services/_shared/personas/code_reviewer/dispatch.py

```
@@ -1628,6 +1629,44 @@ def dispatch_code_review(
+    # Gauge emitted on EVERY review, kills or not (workflow review on PR
+    # #710): a kills-only gauge makes a silently-disabled verifier
+    # indistinguishable from healthy zero-kill traffic - the zero IS the
+    # liveness signal.
+    try:
+        from observability import emit_gauge  # type: ignore
+        emit_gauge(
+            "grug.elder.verification_killed", len(killed),
+            tags={"repo": f"{owner}/{repo_name}"},
+        )
+    except Exception:  # noqa: BLE001 - telemetry never breaks the review
+        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)